### ISSUE TITLE
[Quest][TirisfalGlades] Recruitment

### ISSUE THREAD

**Issue by BraynStorm** [HellscreamWoW/Tracker#2607]
http://www.wowhead.com/quest=26800/recruitment

Most of the corpses on the ground aren't clickable. I've managed to find 5/6 so I can't complete it. I relogged once to see if they will respawn but I wasn't that lucky.

**Comment by Guybrush-Hellscream** [comment]
You can finish this quest, i did it 2 days ago BUT you're quite true : some corpses aren't clickable.

**Comment by BraynStorm** [comment]
I can't find the last one :(. I just abandoned it and they don't even respawn.

**Comment by Guybrush-Hellscream** [comment]
Hmm that's weird, i found 2 in houses, 1 in forge and 3 on the street.

**Comment by BraynStorm** [comment]
I didn't find any in the forge

**Comment by Guybrush-Hellscream** [comment]
Maybe that's the one you're missing. The server didn't restart since 2 days 13 hours, maybe you're right about the fact that you need a server restart to get it spawn again. Worth noticing for devs to fix it.

**Comment by BraynStorm** [comment]
Are you able to issue a server restart? But nahh... it's a small problem to restart the whole server for.

**Comment by Guybrush-Hellscream** [comment]
Yeah, anyway corpses are buggy so that's basically where the trouble lies!

**Comment by BraynStorm** [comment]
I know why they are not clickable. I just logged in I had got the quest form yesterday but Darnell (the guy that spawns when you get the quest) wasn't here. I tried clicking on corpses and they didn't dissappear but were left unclickable. After I retook the quest, every other corpse was clickable,

**Comment by Guybrush-Hellscream** [comment]
I had the guy when i did it and every corpses weren't clickable, that's weird... anyway glad you could finish it at least!

**Comment by BraynStorm** [comment]
Well I took the time and walked around the whole area, every corpse worked. Someone before you must've click on some corpses without the quest or darnell. I hope this info helps the devs

**Comment by MirrorWraith** [comment]
also please make this quest blizzlike, the picked corpse is not on darrel's back, but beneath his feet
thanks
